projects/core/src/cart/facade/active-cart.service.ts

```diff
@@ -391,6 +391,7 @@ export class ActiveCartService implements OnDestroy {
    * @param quantity
    */
   addEntry(productCode: string, quantity: number): void {
+    // TODO: Support multiple, simultaneous invocation of this function, when cart is not loaded/created
```

Comment:
can we create a ticket (explain use case for this improvement -> clicking in the UI 2 times add to cart without modals etc.) and reference it in the TODO #xxx?